docs/api/proposed_endpoints.md

```diff
@@ -0,0 +1,153 @@
+[TOC]
+
+# Posts related actions
+
+## Create a post
+
+Create a post of contents (text + image).
```

Comment:
You cannot give text/image in a single call.

Image will be read chunk by chunk as a streaming HTTP request.

Reply:
Okay, so is that mean we can not create a post with both text and image in it? Sorry if I misunderstood anything.

Reply:
The image goes in a separate request. I've explained into @storymode7 just now.